robot_assignments/web/JetAutomateExercise/testcases/testDoppio.robot

```diff
@@ -0,0 +1,64 @@
+*** Settings ***
+Resource    ${CURDIR}/../import.robot
+
+
+*** Variable ***
+${urlLogin}    http://125.26.15.143:13132/ 
+${username}     oz4899
+${password}     1234
+${username_wrong}   abcd
+${searchText}   shirt
+${searchText_wrong}   abcd
+
+*** Test Cases ***
+Testcase 01 - Verify that user cannot login with wrong username
+    Open Website Doppio
+    Login With Username And Password  ${username_wrong}    ${password}
+    Verify Login failed
+    Close Browser
+
+Testcase 02 - Verify that user can login to the system when username are correct
+    Open Website Doppio
+    Login With Username And Password   ${username}    ${password}
+    Verify Login Pass
+    Close Browser
+
+Testcase03 - Verify that search function are working correctly
+    Open Website Doppio
+    Login With Username And Password   ${username}    ${password}
+
+    Wait Until Element Is Visible       ${searchFieldXpath} 
+    Input Text To Search Bar      ${searchText_wrong}
+    Click Search 
+    Wait Until Page Does Not Contain Element        ${itemCardXpath}
+    Wait Until Element Is Visible       ${notFoundXpath}
+    Verify No Result Found
+
+    Input Text To Search Bar     ${searchText}
```

Comment:
2 step นี้สามารถยุบเป็น 1 feature ได้
kw: search keyword via search bar
steps:
Input Text To Search Bar     ${searchText}
Click Search